DataScience/DataScience.py

```diff
@@ -34,6 +34,10 @@
         print("Start and end dates are expected. Example: python datascience.py 20161122 20161130")
     start_date_string = sys.argv[1]
     start_date = date(int(start_date_string[0:4]), int(start_date_string[4:6]), int(start_date_string[6:8]))
+    
+    # We will populate some data structures with info from previous day
+    start_date_prev = start_date + timedelta(days = -1)
```

Comment:
not sure what you mean by deployment dependent ?  what kind of parameter do u have in mind .. something like 'usePrevDay=True (default)' ?